### PR TITLE
Add LinkedIn link / Styles adjustments: newsletter button and contact

### PR DESCRIPTION
**Pull request by mariana-caldas**
|                                    Web Dev Path                                    |
| :--------------------------------------------------------------------------------: |

#### Have you updated the CHANGELOG.md file? If not, please do it.

Yes.

#### What is this change?

We're adding the LinkedIn link to the footer and also adjusting the newsletter button styles as well as the contact page height on desktop.

#### Were there any complications while making this change?

No.

#### How did you verify this change?

Locally.

#### When should this be merged?

After review.

### CODE REVIEW

File: styles/SubmitButton.module.scss [hunk 1]
```diff
@@ -40,8 +40,10 @@
       border: 1px solid $dark-bg-color;
     }
 
-    @include desktop {
+    @include large-desktop {
```
Comment (cherylli) [inline styles/SubmitButton.module.scss:43]: Not related to your changes but I see in the same file under .btn there's 2 "min-width"
image

**Review by cherylli — APPROVED** [review]

File: styles/SubmitButton.module.scss (hunk 1 above)
Comment (mariana-caldas) [reply styles/SubmitButton.module.scss:43]: The first media-query only refers to the .btn, @cherylli , while the second one only applies to .newsletter__button, so there is no redundancy here. Please let me know if that is what you were talking about, Thanks 🙌🏼 

image

image

Comment (cherylli) [reply styles/SubmitButton.module.scss:43]: I mean line 6 and line 14

Comment (mariana-caldas) [reply styles/SubmitButton.module.scss:43]: Oh, I see now, @cherylli ! My bad!
Making a note to fix that next PR. I will make a PR release to 1.2.0 in the next few days before we start thinking about the blog.

**Pull request merged**